project_name/settings.py

```diff
@@ -428,3 +431,363 @@
             "handlers": ["console"], "level": "DEBUG", },
         },
     }
+
+############################################
+# other settings specific to WorldMap CAMP #
+############################################
+
+# SECRET_KEY = '************************'
+# Make this unique, and don't share it with anybody.
+SECRET_KEY = os.getenv('SECRET_KEY', "{{ secret_key }}")
+
+# per-deployment settings should go here
+SITE_HOST_NAME = os.getenv('SITE_HOST_NAME', "localhost")
+SITE_HOST_PORT = os.getenv('SITE_HOST_PORT', "8000")
+SITEURL = os.getenv('SITEURL', "http://%s:%s/" % (SITE_HOST_NAME, SITE_HOST_PORT))
+
+#
+# General Django development settings
+#
+
+#Define email service on GeoNode
+EMAIL_ENABLE = True
+
+if EMAIL_ENABLE:
+    EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+    EMAIL_HOST = 'smtp.zju.edu.cn'
+    EMAIL_PORT = 25
+    EMAIL_HOST_USER = 'camp2018@zju.edu.cn'
+    EMAIL_HOST_PASSWORD = 'zjuworldmap120!'
```

Comment:
I'd recommend to use env variables for all of these sensible parameters, specially for the password.
For example:
EMAIL_HOST_PASSWORD = os.getenv('EMAIL_HOST_PASSWORD', default='')